modules/core/23-commitment/types/merkle_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	storetypes "cosmossdk.io/store/types"
 
 	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types"
-	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
+	v2 "github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
```

Comment:
many linters like adding this for some reason :smile: 

```suggestion
	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
```